jdisc_http_service/src/main/resources/configdefinitions/jdisc.http.connector.def

```diff
@@ -41,6 +41,24 @@ tcpKeepAliveEnabled                 bool     default=false
 # Enable/disable TCP_NODELAY (disable/enable Nagle's algorithm).
 tcpNoDelay                          bool     default=true
 
+# Whether to enable connection throttling. New connections will be dropped when a threshold is exceeded.
+throttling.enabled                  bool     default=false
+
+# Max number of connections.
+throttling.maxConnections           int      default=-1
+
+# Max memory usage in megabytes (totalMemory - freeMemory).
+throttling.maxMemoryUsage           int      default=-1
```

Comment:
I think this should be a percentage of heap given (totalMemory)

Reply:
Fixed